arelle/plugin/validate/NL/PluginValidationDataExtension.py

```diff
@@ -155,6 +164,21 @@ def getNoMatchLangFootnotes(self, modelXbrl: ModelXbrl) -> set[ModelInlineFootno
     def getOrphanedFootnotes(self, modelXbrl: ModelXbrl) -> set[ModelInlineFootnote]:
         return self.checkFootnotes(modelXbrl).orphanedFootnotes
 
+    def getFactLangFootnotes(self, modelXbrl: ModelXbrl) -> defaultdict[ModelInlineFootnote, set[str]]:
+        return self.checkFootnotes(modelXbrl).factLangFootnotes
+
+    @lru_cache(1)
+    def getReportXmlLang(self, modelXbrl: ModelXbrl) -> str:
+        for ixdsHtmlRootElt in modelXbrl.ixdsHtmlElements:
+            for uncast_elt, depth in etreeIterWithDepth(ixdsHtmlRootElt):
+                elt = cast(Any, uncast_elt)
+                if isinstance(elt, (_ElementTree, _Comment, _ProcessingInstruction, EntityBase)):
+                    continue
+                reportXmlLang = elt.get("{http://www.w3.org/XML/1998/namespace}lang")
+                if reportXmlLang:
+                    return reportXmlLang
+        return 'None'
```

Comment:
This is unexpected. Why not return `None`?

Reply:
fixed